libqtile/backend/x11/core.py

```diff
@@ -111,6 +112,7 @@ def __init__(self, display_name: str | None = None) -> None:
             | EventMask.EnterWindow
             | EventMask.LeaveWindow
             | EventMask.ButtonPress
+            | EventMask.PropertyChange
```

Comment:
I don't love this as it means we get all `PropertyNotify` events on the root window. I wanted to only enable this if `x11_fake_transparency` had been set by `Core.setup_listener` is called before the config is loaded.

One approach could be to use a hook which is fired when the config is loaded and subscribe to that.

Reply:
I dig that hook, I don't have any other ideas right now which involve subscribing to it, but it sounds cool.

Reply:
It's not actually necessary here after all.

Reply:
You can use `Core.on_config_load` (ignore this if you found it)